src/api/spec/controllers/request_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe RequestController, type: :controller do
+  render_views # NOTE: This is required otherwise Suse::Validator.validate will fail
+
+  describe '#request_command' do
+    let!(:user) { create(:confirmed_user) }
+    let!(:bs_request) { create(:bs_request) }
+    before do
+      login user
+    end
+
+    context 'successful' do
+      before do
+        post :request_command, params: { id: bs_request.number, cmd: :diff, format: :xml }
+      end
+
+      it { expect(response).to have_http_status(:success) }
+    end
+
+    context 'with diff_to_superseded parameter' do
+      context 'of a not superseded request' do
+        let!(:another_bs_request) { create(:bs_request) }
+
+        before do
+          post :request_command, params: { id: bs_request.number, cmd: :diff, format: :xml, diff_to_superseded: another_bs_request }
+        end
+
+        it { expect(response).to have_http_status(:not_found) }
+      end
+
+      context 'of a superseded request' do
+        let!(:superseded_bs_request) { create(:bs_request) }
+
+        before do
+          superseded_bs_request.state = :superseded
+          superseded_bs_request.superseded_by = bs_request.number
+          superseded_bs_request.save
```

Comment:
:thought_balloon: In one line: 

`superseded_bs_request.update(superseded_by: bs_request.number, state: :superseded)`